### PR TITLE
Develop27004

### PR DESCRIPTION
**Pull request by AkshayChouguleac1**

### CODE REVIEW

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ModuleName.java [hunk 1]
```diff
@@ -71,13 +71,15 @@ public enum ModuleName {
...
 	WORKFLOW_ACTION_SERVICE,
 
 	WORKFLOW_ACTION_API,
 
-	WORKFLOW_ACTION_JOB;
+	WORKFLOW_ACTION_JOB,
+	
+	REQUEST_HANDLER_SERVICE,
+
+	MESSAGE_SENDER_API;
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ModuleName.java]: Please change to USER_NOTIFICATION_API

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformConstants.java [hunk 2]
```diff
@@ -138,6 +138,9 @@ public final class PlatformConstants {
 	public static final String RPR_WORKFLOW_ACTION_JOB = RPR_REGISTRATION_PROCESSOR_PREFIX + "WAJ-";
 
 	public static final String RPR_FINALIZATION_STAGE = RPR_REGISTRATION_PROCESSOR_PREFIX + "FIN-";
+	
+	public static final String RPR_USER_NOTIFICATION_API = RPR_REGISTRATION_PROCESSOR_PREFIX + "MSA-";
+
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformConstants.java]: Please change MSA to UNA

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java [hunk 3]
```diff
@@ -1401,6 +1401,19 @@ public enum PlatformErrorMessages {
 	/** The queue json validation failed. */
 	QUEUE_JSON_VALIDATION_FAILED(PlatformConstants.RPR_MA_MIDDLEWARE + "004",
 			"Unable to find Connection Properties"),
+	
+	RPR_MAS_MISSING_INPUT_PARAMETER(PlatformConstants.RPR_USER_NOTIFICATION_API + "001",
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java]: Please change MSA to UNA in all error codes

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java [hunk 4]
```diff
@@ -172,7 +172,10 @@ public enum PlatformSuccessMessages {
 			"Packet processing completed with reject status without Parent flow"),
 
 	PACKET_ANONYMOUS_PROFILE(PlatformConstants.RPR_CAMEL_BRIDGE_MODULE + "008",
-			"Packet anonymous profile flow initiated");
+			"Packet anonymous profile flow initiated"),
+	
+	RPR_MESSAGE_SENDER_API_SUCCESS(PlatformConstants.RPR_USER_NOTIFICATION_API + "000",
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java]: RPR_MESSAGE_SENDER_API_SUCCESS to RPR_USER_NOTIFICATION_API_SUCCESS

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java [hunk 5]
```diff
@@ -33,7 +34,9 @@ public static void main(String[] args) {
 		workflowInternalActionVerticle.deployVerticle();
 		WorkflowActionApi workflowActionApi = ctx.getBean(WorkflowActionApi.class);
 		workflowActionApi.deployVerticle();
-		WorkflowActionJob workflowActionJob = ctx.getBean(WorkflowActionJob.class);
-		workflowActionJob.deployVerticle();
+//		WorkflowActionJob workflowActionJob = ctx.getBean(WorkflowActionJob.class);
```
Comment (sowmya695) [inline registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java]: Please uncomment this 37 and 38 lines to execute jobs

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties [hunk 6]
```diff
@@ -1,17 +1,23 @@
 spring.cloud.config.uri=http://localhost:51000/config
 spring.cloud.config.label=master
 spring.application.name=application,registration-processor
-spring.profiles.active=mz
+spring.profiles.active=default
```
Comment (sowmya695) [inline registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties]: Please change to spring.profiles.active=mz

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties [hunk 7]
```diff
@@ -1,17 +1,23 @@
...
 spring.cloud.config.name=registration-processor
 management.endpoints.web.exposure.include=refresh
 registration.processor.zone=secure
 vertx.cluster.configuration=${spring.cloud.config.uri}/*/${spring.profiles.active}/${spring.cloud.config.label}/hazelcast_${registration.processor.zone}.xml
 #management.security.enabled=false
 #disabling health check so that client doesnt try to load properties from sprint config server every
 # 5 minutes (should not be done in production)
+#MessageSender api id
+mosip.regproc.user.notification.api.id=mosip.registration.processor.user.notification
```
Comment (sowmya695) [inline registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties]: Please remove mosip.regproc.user.notification.api.id and version from bootstrap and add in mosip-config

**Review by sowmya695 — COMMENTED** [review]
Please add the test class for UserNotificationAPI with websub changes also

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java [hunk 8]
```diff
@@ -35,5 +36,7 @@ public static void main(String[] args) {
 		workflowActionApi.deployVerticle();
 		WorkflowActionJob workflowActionJob = ctx.getBean(WorkflowActionJob.class);
 		workflowActionJob.deployVerticle();
+		UserNotificationApi messageSenderApi = ctx.getBean(UserNotificationApi.class);
```
Comment (sowmya695) [inline registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java:39]: Please change object name 
messageSenderApi to userNotificationApi

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ModuleName.java (hunk 1 above)
Comment (AkshayChouguleac1) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ModuleName.java]: done

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformConstants.java (hunk 2 above)
Comment (AkshayChouguleac1) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformConstants.java]: done please check earliest commit

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java (hunk 3 above)
Comment (AkshayChouguleac1) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java]: done, please check earliest commit

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java (hunk 4 above)
Comment (AkshayChouguleac1) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformSuccessMessages.java]: done, please check earliest commit

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java (hunk 5 above)
Comment (AkshayChouguleac1) [reply registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java]: done ,please check earliest commit

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties (hunk 6 above)
Comment (AkshayChouguleac1) [reply registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties]: done ,please check earliest commit

File: registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties (hunk 7 above)
Comment (AkshayChouguleac1) [reply registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties]: done ,please check earliest commit

**Review by sowmya695 — APPROVED** [review]

**Pull request merged**